baip-0002.md

```diff
@@ -0,0 +1,62 @@
+    BAIP: 0002
+    Title: Reform of loopholes in feed price mechanism
+    Authors: cn-vote  bitshareschina@163.com
+    Status: Draft
+    Type: Consensus 
+    Created: 2019-11-5
+
+
+# Abstract
+This BAIP defines reforms to the current feed price mechanism vulnerability. The specific program is: The` feed price` is the highest between the `current price` and the ` two-day moving average price` .
```

Comment:
Only one vulnerability reform? The title refers to multiple vulnerabilities (loophole**s**) being reformed by this BAIP?